ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/TopicStream.java

```diff
@@ -187,10 +187,11 @@ boolean isFormat(
       return false;
     }
 
-    String print(ConsumerRecord<String, Bytes> record) throws IOException {
-      String key = record.key() != null ? record.key() : "null";
+    String print(final ConsumerRecord<String, Bytes> record) throws IOException {
+      final String key = record.key() != null ? record.key() : "NULL";
+      final String value = record.value() != null ? record.value().toString() : "NULL";
```

Comment:
nit: could use String.valueOf here. It will take care of the null-check for you. It returns "null" for null values though.

Reply:
Good point. I would use `String.valueOf`, but as nick said, `NULL` would be preferable in this case.

Reply:
`String.valueOf(..).toUpperCase()` ?

Reply:
This would never print `NULL` sine before calling this method you check for `record.value()` to be not null.

Reply:
Agreed. NULL would never be printed given the change in TopicStream on line 67. But still good to have a null check in case anything changes up stream.